grind/bin/grind

```diff
@@ -355,6 +356,11 @@ class TestRunner:
                             type=int,
                             default=default_timeout,
                             help="Per-task timeout. The default timeout is %s seconds." % default_timeout)
+        parser.add_argument('--java-version',
+                            type=int,
+                            help="Select Java version. Valid choices include 7 and 8. " +
```

Comment:
We can validate with argparse's choices, e.g.

parser.add_argument("-v", "--verbosity", type=int, choices=[0, 1, 2],
                    help="increase output verbosity")

Can also set a default value.